app/views/admin/registers/_register.html.haml

```diff
@@ -1,6 +1,4 @@
-%tr
-  %td
-    = register.name
-  %td= register.register_phase
-  %td
-    = link_to 'Edit', edit_admin_register_path(register)
+%li{id: dom_id(register)}
```

Comment:
can we add some explanation e.g. Drag to re-order registers by popularity? Can we add some icon to make it clearer these are draggable?